test/vmtester/vmtester.cpp

```diff
@@ -3,29 +3,18 @@
 // Licensed under the Apache License, Version 2.0.
 
 #include "vmtester.hpp"
-
 #include <evmc/loader.h>
-
+#include <evmc/evmc.hpp>
```

Comment:
(includes look not ordered, "evmc" < "loader")

Reply:
I noticed, but clang-format prefers `.h` over `.hpp` in current configuration.